content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Pod Eviction
+content_type: concept
+weight: 10
```

Comment:
Instead of adding a new page, can we reuse the eviction policy page which has a weight of 60?